### ISSUE TITLE
[Audit Report] [N4] [Suggestion] TODO label

### ISSUE THREAD

**Issue by orionstardust** [aavegotchi/aavegotchi-realm-diamond#18]
### Description
The comment of the alchemicaRecipient and claimAvailableAlchemica functions in the AlchemicaFacet contract have a TODO tag that have not been implemented.

https://github.com/aavegotchi/aavegotchi-realm-diamond/blob/cee38d37307c49dc41cdb737a962d5d313c1cd4f/contracts/RealmDiamond/facets/AlchemicaFacet.sol#L226

https://github.com/aavegotchi/aavegotchi-realm-diamond/blob/cee38d37307c49dc41cdb737a962d5d313c1cd4f/contracts/RealmDiamond/facets/AlchemicaFacet.sol#L243

### Solution
It is recommended to confirm whether the implementation of these functions meets the requirements.

**Comment by orionstardust** [comment]
@cinnabarhorse 
- TODO label in alchemicaRecipient() function is about Aavegotchi lending, so it will be fixed after deployment of Aavegotchi lending.
- TODO label in claimAvailableAlchemica() function is about claimOperator.

I think, they are for future release... Right?

**Comment by cinnabarhorse** [comment]
Yes, that's right. We will remove the todo labels once they are finished.

**Comment by cinnabarhorse** [comment]
We'll leave this issue open until the todo labels are removed.
